src/Microsoft.Health.SqlServer/Features/Schema/Model/ColumnUtilities.cs

```diff
@@ -21,7 +22,7 @@ internal static long GetLengthForFloatColumn(byte precision)
         }
         else
         {
-            throw new ArgumentOutOfRangeException(string.Format("Precision {0} must be between 1 & 53", precision));
+            throw new ArgumentOutOfRangeException(string.Format(CultureInfo.CurrentCulture, "Precision {0} must be between 1 & 53", precision));
```

Comment:
Why `CurrentCulture` in some places and `InvariantCulture` in others?

Reply:
I've updated the cultures such that `CurrentCulture` is used for client-facing messages, like exception messages, while `InvariantCulture` is used for our own log messages and parsing.